api/src/main/java/edu/cornell/mannlib/vitro/webapp/rdfservice/impl/jena/tdb/RDFServiceTDB.java

```diff
@@ -93,6 +93,10 @@ public boolean changeSetUpdate(ChangeSet changeSet)
 		}
 	}
 
+        @Override
+        public boolean preferPreciseOptionals() {
+            return true;
```

Comment:
@brianjlowe Does this change the default for TDB? In my (limited) testing using the precise subqueries improved performance with TDB, though as I understood it that was unexpected.

Reply:
That's right; this changes the default for TDB.  The suggestion on the last call was that we should optimistically switch the TDB default, based on the limited results you and I had, and switch back if more extensive testing of the release candidate suggested otherwise.